src/dmqproto/client/mixins/NeoSupport.d

```diff
@@ -136,10 +136,8 @@ template NeoSupport ( )
             ( cstring channel, Consume.Notifier notifier, Options options )
         {
             cstring subscriber;
-            setupOptionalArgs!(options.length)(
-                options,
-                (Subscriber sub) {subscriber = sub.name;}
-            );
+            scope dg =  (Subscriber sub) { subscriber = sub.name; };
```

Comment:
Suggest changing `dg` to `parse_subscriber`, in line with what's being discussed in dlsproto.

Reply:
Fixed